<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edca_config</class>
 <widget class="QDialog" name="Edca_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>AC_VO</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cw_min</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cw_max</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TXOP_LIMIT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>AIFSN</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>7</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="value">
       <number>47</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>AC_VI</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cw_min</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>7</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>15</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_8">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>94</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_9">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cw_max</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>AIFSN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TXOP_LIMIT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>AC_BE</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cw_min</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>65532</number>
      </property>
      <property name="value">
       <number>15</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_10">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>1023</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_11">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_12">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cw_max</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>AIFSN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TXOP_LIMIT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>AC_BK</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cw_min</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_13">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>15</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_14">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>1023</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_15">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>7</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_16">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cw_max</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>AIFSN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TXOP_LIMIT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>AMsdu</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>AMsdu</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>72</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>AC_VO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AC_VI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AC_BE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AC_BK</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="spinBox_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>65535</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Max AMsdu Size</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>AMpdu</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>AMpdu</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>72</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>AC_VO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AC_VI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AC_BE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AC_BK</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Max AMsdu Size</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>65535</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Density(us)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>40</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Edca_config</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Edca_config</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
